css/style.css

```diff
@@ -0,0 +1,397 @@
+.m-2 {
+    margin: 2rem;
+}
+
+.ml-1 {
+    margin-left: 1rem;
+}
+
+.ml-2 {
+    margin-left: 2rem;
+}
+
+.mr-1 {
+    margin-right: 1rem;
+}
+
+.mr-2 {
+    margin-right: 2rem;
+}
+
+.mt-1 {
+    margin-top: 1rem;
+}
+
+.mt-2 {
+    margin-top: 2rem;
+}
+
+.mb-1 {
+    margin-bottom: 1rem;
+}
+
+.mb-2 {
+    margin-bottom: 2rem;
+}
+
+/* end of layout */
+
+/* html */
+
+html {
+    font-family: sans-serif;
+    -ms-text-size-adjust: 100%; 
+    -webkit-text-size-adjust: 100%; 
+}
+html {
+    display: block;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+/* end of html */
+
+/* head */
+
+head {
+    display:none;
+}
+
+/* end of head */
+
+/* body */
+
+body {
+    margin: 0;
+    display: block;
+}
+
+/* end of body */
+
+/* header */
+
+.header-image {
+    padding: 2rem;
+    margin-bottom: 1rem;
+    background: #3a2929;
+    display: flex;
+    flex-wrap: wrap;
+} 
+
+.w-100 {
+    width: 5%;
+}
+
+.numb1 {
+    display: flex;
+    justify-content: flex-end;
+    flex-basis: 88.666667%;
+    max-width: 88.666667%;
+}
+    
+.number {
+    padding: 1rem;
+    background-color: #ad8383;
+}
+/* end of header */
+
+/* nav bar */
+
+.navbar {
+    padding-left: 1rem;
+    padding-right: 2rem;
+    padding-top:   1rem;
+    padding-bottom: 2rem;
+    margin-bottom: 1rem;
+    background: #ad8383;
+}
+
+.nav-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+ul {
+    list-style-type: none;
+    padding-left: 0;
+}
+
+ul li a {
+    text-decoration: none;
+}
+
+/* end of nav bar */
+
+/* main:section 1 */
+
+#info .row {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.info {
+    padding: 2rem;
+    flex-basis: 33.333333%;
+    max-width: 33.333333%;
+    background: #d1cecd;
+}
+
+.text {
+    margin-bottom: 2rem;
+
+}
+
+.sec-1a {
+    flex-basis: 66.666667%;
+    max-width: 66.666667%;
+}
+
+.carousel {
+    padding: 2rem;
+    margin-left: 2rem;
+    background: #d1cecd;
+    
+}
+
+.carousel img {
+    width: calc(100% - 2rem);
+}
+
+.row-1 {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 32px;
+}
+
+
+/* end main:section 1 */
+
+/* main:section 2 */
+
+.row2 {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.row2 img {
+    width: 100%;
+}
+
+.sec2 {
+    margin: 1rem;
+    margin-top: 2rem;
+}
+
+.cakes-sec2 {
+    flex-basis: 25%;
+    max-width: 25%;
+}
+/* end of main:section 2 */
+
+/* footer */
+
+.footer {
+    padding: 2rem;
+    background: rgb(160, 122, 122);
+}
+
+.footer2 {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+/* end of footer */
+
+/* @media screen */
+
+@media only screen and (max-width:350px) {
+    image {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: column-reverse;
+        align-content: center;
+    }
+
+
+ /*  For devices smaller than 350px: */ 
+   
+  body {
+    background-image: url('images/cake-1.jpeg' 'images/cakes-2.jpeg' 'images/cakes-3.jpeg' 'images/cake-4.jpeg');
```

Comment:
The background image only takes one argument so this css statement is currently broken and won't be applied. 

i.e it should be background-image: url('images/cake-1.jpeg') if you want the first image to cover the whole body background